frontend/src/components/Maps.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component, PropTypes } from 'react'
+import Header from './Header.js'
+import MapperCard from './MapperCard.js'
+import MapCard from './MapCard.js'
+import MapListItem from './MapListItem.js'
+
+class Maps extends Component {
+  render = () => {
+    const { maps, currentUser, section, displayStyle, user, moreToLoad, loadMore } = this.props
+    let mapElements
+
+    if (displayStyle == 'grid') {
+      mapElements = maps.models.map(function (map) {
+        return <MapCard key={ map.id } map={ map } currentUser={ currentUser } />
+      })
+    }
+    else if (displayStyle == 'list') {
+      mapElements = maps.models.map(function (map) {
+        return <MapListItem key={ map.id } map={ map } />
```

Comment:
see my other comment... leaving placeholder for unimplemented thing :p is that a terrible idea? It's kind of motivating!